_sidebar.rst.inc

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I don't think this file should be here

Reply:
Removed and added to gitignore to avoid further problems